README.md

```diff
@@ -11,21 +11,23 @@ This repository allows you to spin up a local test environment for the ``sdx-``
  - python3
 
 ### Getting started
+Make a directory called `sdx` e.g. `/users/[your_name]/sdx`.
+
+This will contain all the files for the sdx services,
 
-Export a ``SDX_HOME`` environment variable. This should point at a folder into
-which you wish to clone the rest of the services (e.g. ``/home/my-user/sdx``)
 [see [configuration](#configuration)]
 
-Set up a virtual environment using Python3 and ensure you activate it.
+Open your .bash_profile and add the line: ``export SDX_HOME=[LOCATION OF SDX FOLDER]``
+
+Apply these changes to your bash_profile by doing ``~/.bash_profile``
```

Comment:
This line is wrong, the command is:
`. ~/.bash_profile`

Also, why are you using pairs of backticks?  Use either 1 or 3